tests/robot/resources/variables.robot

```diff
@@ -3,4 +3,7 @@
 @{ENCLAVE_SUBDOMAINS}               edi  edge  airflow  flower
 ${TEST_MODEL_ID}                    demo-abc-model
 ${TEST_MODEL_1_VERSION}             1.0
-${TEST_MODEL_2_VERSION}             1.1
```

Comment:
Looks like this modifications (entire file) are obsolete

Reply:
the entire file matches the same in develop